.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+on: [pull_request]
+
+env:
+  NODE_VERSION: 20.11.0
```

Comment:
it should be mentioned that all admin panel team works with nodejs v.18 . Maybe it must be specified in package.json and here as well to reduce possible errors relating to different versions of nodejs

Reply:
Thanks for clearing that up. I'll revise the node version